_data/team_members.yml

```diff
@@ -1,5 +1,5 @@
 - name: Satrajit S Ghosh
-  photo: //satra.cogitatum.org/assets/satra.png
+  photo: /assets/team/Satrajit_S_Ghosh.png
```

Comment:
i think these will need `../assets/...`

Reply:
I can do that too  ... done in 9ecc62410ba81c4512130361673c446d9ea8862a . I guess we will know for sure if we try